cmd/node/factory/structs.go

```diff
@@ -780,6 +785,26 @@ func ProcessComponentsFactory(args *processComponentsFactoryArgs) (*Process, err
 	}, nil
 }
 
+func applyblockTrackerOnInterceptors(
+	finalityAttester process.FinalityAttester,
+	interceptorsContainer process.InterceptorsContainer,
+) error {
+
+	var foundErr error
+	interceptorsContainer.Iterate(func(key string, interceptor process.Interceptor) bool {
+		dataFactory := interceptor.InterceptedDataFactory()
+		setter, ok := dataFactory.(interceptedDataFactoryWithFinalAttesterSetter)
+		if ok {
```

Comment:
if !ok return true else setter.set and return foundErr == nil. If you have a set of errors, you only catch the last one. Should save founderr into an other variable.

Reply:
function removed, no longer necessary